### DISCUSSION TITLE
Error cs0433

### DISCUSSION THREAD

**Discussion by CleitonTito** [balta-io/2811#22] [Q&A] [answered]
Olá Balta, tudo bem?
Estou no módulo de autenticação e antes de rodar o projeto após ter incluso o AddTransient resolvi rodar o dotnet build para saber se havia erros, e está me apontando um erro cs0433, em que fala que há duas descrições iguais para o IdentityModel.Tokens, porem com versões do dotnet diferentes, uma para o 6.10 e a outra para o 3.5, sendo que eu nunca instalei a versão 3.5.
Abri powershell para conferir, e eu realmente só tenho a versão 5 e 6 do dotnet.

Poderia me dar um apoio?

Obrigado desde já
![Erro build](https://user-images.githubusercontent.com/81769859/171266820-5fd420dd-5d79-4555-9aac-b182433869ea.jpg)
![List runtimes](https://user-images.githubusercontent.com/81769859/171266832-9e9b2753-f0c8-41e9-b627-a8cd29008c01.jpg)

**Comment by andrebaltieri** [comment, marked as answer]
@CleitonTito consegue colar o conteúdo do arquivo `.csjproj` dos dois projetos aqui?

**Comment by CleitonTito** [comment]
@andrebaltieri, não há mais de 1 projeto.
Segue o conteúdo .csjproj:

<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>net6.0</TargetFramework>
    <Nullable>enable</Nullable>
    <ImplicitUsings>enable</ImplicitUsings>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="6.0.5" />
    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.5" />
    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="6.0.5">
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
      <PrivateAssets>all</PrivateAssets>
    </PackageReference>
    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="6.0.5" />
    <PackageReference Include="Microsoft.IdentityModel" Version="7.0.0" />
  </ItemGroup>

</Project>

**Reply by andrebaltieri** [reply]
@CleitonTito não veio o conteúdo... veio só uma linha...

**Reply by CleitonTito** [reply]
não entendi o porque, mas agora que reparei que mesmo colando tudo o Git esconde o restante, bem, deixarei uma imagem do conteúdo então.
![conteudo csproj](https://user-images.githubusercontent.com/81769859/171644896-92eeca83-922f-4a5f-9c9a-be8508c02007.JPG)
